install/kubernetes/cilium/README.md

```diff
@@ -75,8 +75,11 @@ contributors across the globe, there is almost always someone available to help.
 | bpf.ctAnyMax | int | `262144` | Configure the maximum number of entries for the non-TCP connection tracking table. |
 | bpf.ctTcpMax | int | `524288` | Configure the maximum number of entries in the TCP connection tracking table. |
 | bpf.hostBoot | string | `"/boot"` | Configure the path to the host boot directory |
+| bpf.hostLegacyRouting | bool | `false` | Configure whether direct routing mode should route traffic via host stack (true) or directly and more efficiently out of BPF (false) if the kernel supports it. The latter has the implication that it will also bypass netfilter in the host namespace. |
+| bpf.lbBypassFIBLookup | bool | `true` | Configure the FIB lookup bypass optimization for nodeport reverse NAT handling. |
```

Comment:
This flag has actually already been removed and does not have any effect anymore. Let's remove it from the Helm charts too.

Reply:
ack

Reply:
I see https://github.com/cilium/cilium/blob/master/Documentation/network/kubernetes/kubeproxy-free.rst has `lbBypassFIBLookup`, should that be manually removed too? the whole section?

```
NodePort with FHRP & VPC
When using Cilium's kube-proxy replacement in conjunction with a [FHRP](https://en.wikipedia.org/wiki/First-hop_redundancy_protocol) such as VRRP or Cisco's HSRP and VPC (also known as multi-chassis EtherChannel), the default configuration can cause issues or unwanted traffic flows. This is due to an optimization that causes the source IP of ingress packets destined for a NodePort to be associated with the corresponding MAC address, and later in the reply, the MAC address is used as the destination when forwarding the L2 frame, bypassing the FIB lookup.

In such an environment, it may be preferred to instruct Cilium not to attempt this optimization. This will ensure the response is always forwarded to the MAC address of the currently active FHRP peer, no matter the origin of the incoming packet.

To disable the optimization set bpf.lbBypassFIBLookup to false.

helm install cilium [|CHART_RELEASE|](https://github.com/cilium/cilium/blob/master/Documentation/network/kubernetes/kubeproxy-free.rst#id45) \
    --namespace kube-system \
    --set kubeProxyReplacement=strict \
    --set bpf.lbBypassFIBLookup=false

```

Reply:
I'm not familiar enough with the feature to judge that. Let me ping some folks on this.

Reply:
Answer is in: Yes, let's remove this whole section too.